### DISCUSSION TITLE
osu! and Plank dock

### DISCUSSION THREAD

**Discussion by SIUUUUUUUUUU17** [ppy/osu#30781] [Q&A]
How do we add a shortcut to [Plank dock](https://launchpad.net/plank)?

I'm running a MacOS-esque setup on my Cinnamon install using Plank

![image](https://github.com/user-attachments/assets/a992fdda-fdba-4e19-9edc-99e74cd93af2)

**Comment by 12beesinatrenchcoat** [comment]
Similar to macOS, try right-clicking the osu! icon while it is open and select "Keep in dock", or try dragging the AppImage into the dock. https://wiki.archlinux.org/title/Plank#Adding_new_dock_icons
